packages/office-ui-fabric-react/src/utilities/dragdrop/DragDropHelper.tsx

```diff
@@ -10,7 +10,7 @@ import {
 } from './interfaces';
 import { ISelection } from '../../utilities/selection/interfaces';
 
-const DISTANCE_FOR_DRAG_SQUARED = 25; // the minimum mouse move distance to treat it as drag event
+const DISTANCE_FOR_DRAG_SQUARED = 9; // the minimum mouse move distance to treat it as drag event
```

Comment:
The lower this number goes, the easier it is for there to be accidental drags. Drag/drop which results in a move or other immediate-commit operation can cause user grief.